airbyte-integrations/connector-templates/generator/plopfile.js

```diff
@@ -25,18 +25,42 @@ module.exports = function (plop) {
   const pythonSourceInputRoot = '../source-python';
   const singerSourceInputRoot = '../source-singer';
   const genericSourceInputRoot = '../source-generic';
+  const pythonCdkSourceInputRoot = '../source-python-cdk';
 
   const outputDir = '../../connectors';
   const pythonSourceOutputRoot = `${outputDir}/source-{{dashCase name}}`;
   const singerSourceOutputRoot = `${outputDir}/source-{{dashCase name}}-singer`;
   const genericSourceOutputRoot = `${outputDir}/source-{{dashCase name}}`;
+  const pythonCdkSourceOutputRoot = `${outputDir}/source-{{dashCase name}}`;
```

Comment:
I was toying with making this `${outputDir}/source-{{dashCase name}}-cdk` to differentiate sources build with cdk and without. Do you think that is a useful distinction?